database/src/main/java/database/controller/DatabaseServiceRestController.java

```diff
@@ -122,6 +130,44 @@ void handleAddNewRecord(Context ctx) {
         ctx.json(databaseService.addNewRecordToTable(entity));
     }
 
+    @OpenApi(
+            summary = "Add multiple new records to a table",
+            operationId = "addNewRecords",
+            path = "/api/v1/database/{entityClass}/batch",
+            methods = HttpMethod.POST,
+            pathParams = {
+                    @OpenApiParam(name = "entityClass", description = "Class of the entity", required = true)
+            },
+            requestBody = @OpenApiRequestBody(
+                    content = @OpenApiContent(from = BaseEntity[].class),
+                    description = "List of new records to be added",
+                    required = true
+            ),
+            responses = {
+                    @OpenApiResponse(status = "200", description = "Records added successfully",
+                            content = @OpenApiContent(from = BaseEntity[].class)),
+                    @OpenApiResponse(status = "400", description = "Invalid entity data or empty list"),
+                    @OpenApiResponse(status = "400", description = "Table not found"),
```

Comment:
Please change to `Invalid entity class`, see swagger description in `handleAddNewRecord`.